WTWSMS/history/titles/k_saxony.txt

```diff
@@ -1,13 +1,13 @@
 150.1.1={
-	law = succ_gavelkind 
+	law = succ_gavelkind
 	law = cognatic_succession
 }
 450.1.1={
-	holder = 45
+	holder = 45 # Hadugato, fictional
 }
 497.1.1={
-	holder = 30037 
+	holder = 30037 # Gunther, fictional
```

Comment:
I might be wrong on these, but I think it might be based upon splitting up historical rulers.

Reply:
Done.